src/EnergyPlus/WaterCoils.cc

```diff
@@ -402,18 +402,12 @@ void GetWaterCoilInput(EnergyPlusData &state)
                                                                                       NodeInputManager::compFluidStream::Primary,
                                                                                       ObjectIsNotParent);
 
-        {
-            auto const SELECT_CASE_var(AlphArray(7));
-            if (SELECT_CASE_var == "UFACTORTIMESAREAANDDESIGNWATERFLOWRATE") {
-                state.dataWaterCoils->WaterCoil(CoilNum).CoilPerfInpMeth = state.dataWaterCoils->UAandFlow;
+        if (AlphArray(7) == "NOMINALCAPACITY") { // not "UFACTORTIMESAREAANDDESIGNWATERFLOWRATE"
+            state.dataWaterCoils->WaterCoil(CoilNum).CoilPerfInpMeth = state.dataWaterCoils->NomCap;
```

Comment:
I know this is not part of this PR, but it looks like there are still some `int const`'s lurking in `state`.

Reply:
There are, and I've a way to sniff them out, along with other const Array1D and things, so they'll be fixed separate.  Thanks for noticing!